apps/condo/bin/generate-miniapps.js

```diff
@@ -67,18 +66,19 @@ class AppsGenerator {
                     detailedDescription: faker.lorem.paragraphs(5),
                     contextDefaultStatus: this.withFrames ? 'Finished' : 'InProgress',
                     category: this.category,
-                    appUrl: this.withFrames ? 'http://localhost:3001' : undefined,
+                    isHidden: false, isGlobal: false,
+                    appUrl: this.withFrames ? this.appUrl : undefined,
                 })
             }
         }
     }
 }
 
-const generateMiniapps = async ([category, amount, framed]) => {
+const generateMiniapps = async ([category, amount, framed, appUrl]) => {
```

Comment:
With appUrl framed parameter is redundant

Reply:
yes, but I don't want backward incompatible changes for bin scripts

Reply:
It's better to remove the framed argument if you think changing the API is okay. I can change it if you think it's ok

Reply:
It's internal util used by few of us. I guess it's ok to change it here

Reply:
ok